src/test/java/service/impl/CsvFileWriterServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import service.ReaderDataService;
+import service.WriterDataService;
+
+public class CsvFileWriterServiceTest {
+    private static final String PATH_TO_FILE = "src/test/resources/OutputFile.csv";
+    private static final String INVALID_PATH_TO_FILE = "src/test/resources/invalidPath.csv";
+    private static WriterDataService writerDataService;
+    private static ReaderDataService readerDataService;
```

Comment:
you should not depend on your another service
please read with Files.readAllLines() / Files.readString()